validation/validation_test.go

```diff
@@ -30,6 +30,19 @@ const (
 	GenesisAllocsMetadataTest    = "Genesis_Allocs_Metadata"
 )
 
+type (
+	SubTest         func(t *testing.T)
+	SubTestForChain func(t *testing.T, chain *ChainConfig)
+)
```

Comment:
See other comments, maybe
```suggestion
type (
	SubTest         = func(t *testing.T)
	SubTestForChain = func(t *testing.T, chain *ChainConfig)
)
```
instead?

Reply:
Yeah this is maybe marginally better [90672aa](https://github.com/ethereum-optimism/superchain-registry/pull/552/commits/90672aa6faa305697baabe0b4c5099cfa3322ac5).

Reply:
I mean, if you do this, you don't need to set global function `var`s, but can define normal functions.